concordium-consensus/tests/scheduler/SchedulerTests/SmartContracts/V0/SmartContractTests.hs

```diff
@@ -317,7 +317,10 @@ simpleTransferTestCases =
     contractAddr = Types.AddressContract $ ContractAddress 0 0
 
     -- When using pointing to 32 0s in memory, this is the address that is used.
-    Right accRef = addressFromText "2wkBET2rRgE8pahuaczxKbmv7ciehqsne57F9gtzf1PVdr2VP3"
+    accRef = case addressFromText "2wkBET2rRgE8pahuaczxKbmv7ciehqsne57F9gtzf1PVdr2VP3" of
+        Right ref -> ref
+        -- This should not happen, since addressFromText was passed a valid address string.
+        Left _ -> error "addressFromText did not return an account reference"
```

Comment:
```suggestion
    accRef = fromRight (addressFromText "2wkBET2rRgE8pahuaczxKbmv7ciehqsne57F9gtzf1PVdr2VP3")
```

Reply:
No, my memory failed me is what that is. :P 

You could still do `fromRight (error "This should not happen because I say so.")) w`